app/code/Magento/Bundle/Test/Mftf/ActionGroup/AssertAdminErrorIsNotVisibleActionGroup.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="AssertAdminErrorIsNotVisibleActionGroup">
+        <arguments>
+            <argument name="fieldIndex" type="string"/>
+        </arguments>
+
+        <dontSeeElement selector="{{AdminProductFormBundleSection.fieldError(fieldIndex)}}" stepKey="seeError"/>
```

Comment:
In case the selector applies to all products.
```suggestion
        <dontSeeElement selector="{{AdminProductFormSection.fieldError(fieldIndex)}}" stepKey="seeError"/>
```